lib/structures/Attachment.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+const Base = require("./Base");
+
+/**
+ * Represents an attachment
+ * @prop {Object} data The attachment data
+ * @prop {String} data.id The attachment ID
+ * @prop {String} data.filename The filename of the attachment
+ * @prop {String?} data.description The description of the attachment
+ * @prop {String?} data.contentType The content type of the attachment
+ * @prop {Number} data.size The size of the attachment
+ * @prop {String} data.url The URL of the attachment
+ * @prop {String} data.proxyUrl The proxy URL of the attachment
+ * @prop {Number?} data.height The height of the attachment
+ * @prop {Number?} data.width The width of the attachment
+ * @prop {Boolean?} data.ephemeral Whether the attachment is ephemeral
```

Comment:
What's with all the `data.`s?

Reply:
Oh, it's for a class, I thought it were options for a method or something 💀

Reply:
should be fixed there, I'm just being stupid